ui/plugin-system/src/components/TimeSeriesQueryEditor/TimeSeriesQueryEditor.tsx

```diff
@@ -28,7 +28,7 @@ export function TimeSeriesQueryEditor({ queries = [], onChange }: TimeSeriesQuer
   // Build the default query plugin
   // Use as default the plugin kind explicitly set as default or the first in the list
   const { defaultPluginKinds } = usePluginRegistry();
-  const { data: timeSeriesPlugins } = useListPluginMetadata(TIME_SERIES_QUERY_KEY);
```

Comment:
TimeSeriesQueryEditor and TraceQueryEditor components are veeeery similar so some little changes on that one made it quite generic. Should we think of making a query editor generic?